templates/download/index.twig

```diff
@@ -7,18 +7,18 @@
 		   href="{{ rel.browser_download_url }}">
 			<span id="button-title">LMMS</span><br>
 			<span class="fa fa-download download-icon"></span>
-			<big>Free Download</big><br>
+			<big>{% trans %}Free Download{% endtrans %}</big><br>
 			<small>{{ rel.release.name }}&nbsp;({{ rel.osname }})</small>
 		</a>
 	{% endif %}
 {% endmacro %}
 
 {% macro releasenotes(notes, divid) %}
 <div id="release-notes">
-	<a data-toggle="collapse" href="#collapse_{{ divid }}">Show release notes<br><i class="fa fa-angle-down"></i></a>
+	<a data-toggle="collapse" href="#collapse_{{ divid }}">{% trans %}Show release notes{% endtrans %}<br><i class="fa fa-angle-down"></i></a>
 	<div id="collapse_{{ divid }}" class="panel-collapse collapse">
 		<div class="release-notes well">
-			{{ notes|markdown }}
+			{{ ('|GH|' ~ notes)|markdown }}
```

Comment:
If you still son't understand see [this file](https://github.com/LMMS/lmms.io/pull/209/files#diff-e392982e7dc36ce87286c39bd670ebcf)